docs/source/section.rst

```diff
@@ -0,0 +1,84 @@
+.. section:
+
+=================
+Section Bookmarks
+=================
+
+If you have a long page, and you want to allow the visitors of your site to quickly navigate to
+different sections, then you can use bookmarks and create links to the different sections of an HTML
+page.
+
+When a user clicks on a bookmark link, then that page will load as usual but will scroll immediately,
+so that the bookmark is at the very top of the page. Bookmarks are also known as anchors. They can
+be added to any HTML element using the attribute ``id``. For example:
+
+.. code-block:: html
+
+	<section id="unique-identifier-for-that-page">
+
+For obvious reasons, this identifier must be unambiguously, otherwise the browser does not know
+where to jump to. Therefore **djangocms-cascade** enforces the uniqueness of all bookmarks used on
+each CMS page.
+
+
+Configuration
+=============
+
+The HTML standard allows the usage of the ``id`` attribute on any element, but in practice it only
+makes sense on ``<section>``, ``<article>`` and the various heading elements. Cascade by default is
+configured to allow bookmarks on the **SimpleWrapperPlugin** and the **HeadingPlugin**. This can
+be overridden in the project's configuration settings using:
+
+.. code-block:: python
+
+	CMSPLUGIN_CASCADE = {
+	    ...
+	    'plugins_with_bookmark': [list-of-plugins],
+	    ...
+	}
+
+
+Hashbang Mode
```

Comment:
I don't know what hashbang mode is. It seems to be mostly an Angular thing? I think we should clarify this or provide a link with more information, to avoid confusing the reader.